### ISSUE TITLE
Perera et al, - 2019 - Learning Deep Features for One-Class Classification

### ISSUE THREAD

**Issue by wagaken** [wagaken/PAPPER_SUMMARY#17]
IEEE TRANSACTIONS ON IMAGE PROCESSING

**Comment by wagaken** [comment]
### Introduction

Representation learning for one-class classification

**Problem setting**
ordinary fine-tuning is not applicable to anomaly detection

![image](https://user-images.githubusercontent.com/49052500/151915969-0adcb2a6-9ecd-45c5-9b2c-39f01e9a9ff3.png)

**Approach**
novel architecture using parallel CNN with compactness loss and descriptiveness loss
compactness loss - consistency of feature embedding (intra-class variability)
discriminative loss (cross-entropy) - inter-class similarity with classes in the reference dataset

**Characteristics**
transferable model

### Method
![image](https://user-images.githubusercontent.com/49052500/151914437-4c78bd60-de65-4dd1-96de-3519c93b8d68.png)

use the provided one-class dataset to calculate the compactness loss. An external multi-class reference dataset is used to evaluate the descriptiveness loss

![image](https://user-images.githubusercontent.com/49052500/151915354-d0659920-79c9-4b70-a274-70ea7aea90b4.png)

![image](https://user-images.githubusercontent.com/49052500/151916627-d416a15e-1f48-4b4a-b22f-2935f437ebb3.png)

**compactness loss**
![image](https://user-images.githubusercontent.com/49052500/151918100-9512f523-37f1-44ca-902f-d88a51d7f06c.png)

![image](https://user-images.githubusercontent.com/49052500/151918121-79269dc7-4c4e-467d-bdb2-58e5120b4b4a.png)

**total loss**
![image](https://user-images.githubusercontent.com/49052500/151917060-50999089-5a0a-4337-9d8d-5a228d32db74.png)

###  Experimental result

![image](https://user-images.githubusercontent.com/49052500/151917724-7d79bea8-763b-49c7-89bc-4c74a387612c.png)

![image](https://user-images.githubusercontent.com/49052500/151917764-15d16872-65d3-4b1d-b8b5-6ecf1ddf69ee.png)

**Comment by wagaken** [comment]
### Related work
Feature extractor + One-class classifier
Q. Leng, H. Qi, J. Miao, W. Zhu, and G. Su, “One-class classification with extreme learning machine,” Math. Problems Eng., vol. 2015, Nov. 2015, Art. no. 412957

Continuous authentication in mobile devices
U. Mahbub, S. Sakar, V. Patel, and R. Chellappa, “Active user authentication for smartphones: A challenge data set and benchmark results,” in Proc. IEEE 8th Int. Conf. Biometrics Theory, Appl. Syst. (BTAS), Sep. 2016, pp. 1–8.

P. Samangouei and R. Chellappa, “Convolutional neural networks for attribute-based active authentication on mobile devices,” in Proc. IEEE Int. Conf. Biometrics, Theory, Appl. Syst., Sep. 2016, pp. 1–8.
